### PR TITLE
hostdb: add test; `brew audit --strict`

### PR DESCRIPTION
**Pull request by L2G**

### CODE REVIEW

**Comment by DomT4** [comment]
@BrewTestBot test this please

File: Library/Formula/hostdb.rb [hunk 1]
```diff
@@ -1,14 +1,23 @@
...
-  homepage 'http://code.google.com/p/hostdb/'
-  url 'https://hostdb.googlecode.com/files/hostdb-1.004.tgz'
-  sha1 '65ec59c2c88b763813fa611d8fd28a45cd9d5278'
+  homepage "https://code.google.com/p/hostdb/"
+  url "https://hostdb.googlecode.com/files/hostdb-1.004.tgz"
+  sha256 "beea7cfcdc384eb40d0bc8b3ad2eb094ee81ca75e8eef7c07ea4a47e9f0da350"
+
+  def examples_path
+    share + "examples" + name
+  end
```
Comment (mikemcquaid) [inline Library/Formula/hostdb.rb]: Just inline this function, thanks.

Comment (L2G) [inline Library/Formula/hostdb.rb]: You may change your mind again after seeing what had to be done to keep it passing `brew audit --strict`. Just let me know. :smile_cat:

File: Library/Formula/hostdb.rb [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+  url "https://hostdb.googlecode.com/files/hostdb-1.004.tgz"
+  sha256 "beea7cfcdc384eb40d0bc8b3ad2eb094ee81ca75e8eef7c07ea4a47e9f0da350"
 
   def install
-    bin.install Dir['bin/*']
-    doc.install Dir['docs/*']
-    (share+'examples'+name).install Dir['examples/*']
+    bin.install Dir["bin/*"]
+    doc.install Dir["docs/*"]
+    (share + "examples" + name).install Dir["examples/*"]
```
Comment (mikemcquaid) [inline Library/Formula/hostdb.rb]: No need to change spacing here: `share+"examples"+name`

File: Library/Formula/hostdb.rb [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
+  end
+
+  test do
+    system("#{bin}/mkzones -z " +
```
Comment (mikemcquaid) [inline Library/Formula/hostdb.rb]: Interpolate the strings and separate arguments e.g. `system "#{bin}/mkzones", "-z", "#{share}/examples/#{name}//example1/zoneconf.txt`

**Comment by dunn** [comment]
Merged in 96406ea7d2068e749ec4c5d8bf3ade73c03fd592, thanks again!

**Pull request closed without merging**